image/image.go

```diff
@@ -323,9 +328,13 @@ func bootstrapToRootDir(tsto *ToolingStore, model *asserts.Model, opts *Options,
 
 	seen := make(map[string]bool)
 	var locals []string
-	downloadedSnapsInfo := map[string]*snap.Info{}
+	downloadedSnapsInfoForBootConfig := map[string]*snap.Info{}
 	var seedYaml snap.Seed
 	for _, snapName := range snaps {
+		if snapName == "" {
```

Comment:
if we want this check, but see comment below, it should happen earlier than here I think

Reply:
I removed this from this PR again and moved the capturing of baseName some lines earlier.